vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -218,15 +224,13 @@ components:
     CpuConfig:
       required:
       - boot_vcpus
+      - max_vcpus
```

Comment:
Please simply remove this as it's not needed.

Reply:
Nevermind this comment. We need to add some default to serde, but this will be done as a follow up PR.